Source/ThirdParty/WPE-platform/src/westeros/WesterosViewbackendInput.cpp

```diff
@@ -296,6 +318,7 @@ void WesterosViewbackendInput::pointerHandleAxis( void *userData, uint32_t time,
         struct wpe_input_axis_event event{ wpe_input_axis_event_type_motion, e->time, coords.first, coords.second, e->axis, -wl_fixed_to_int(e->value) };
         wpe_view_backend_dispatch_axis_event(backend_input.m_viewbackend, &event);
 
+        g_ptr_array_remove_fast(backend_input.m_axisEventDataArray, data);
```

Comment:
why using g_ptr_array_remove_fast instead of g_ptr_array_remove? I believe it introduces a small memory leak in long-running scenarios